en/documentation/index.md

```diff
@@ -144,9 +149,9 @@ Here is a list of popular tools used by Rubyists:
 
 ### Further Reading
 
-[Ruby-Doc.org][34] maintains a comprehensive list of English
-documentation sources. There are also plenty of [books about Ruby][35].
-If you have questions about Ruby the [mailing
+[Ruby-Doc.org][34] and [Omniref.com][40] both maintain a comprehensive 
```

Comment:
The paragraph is refering to that list, not to the API documentation (so it's no double standard).

Granted, the list is not up-to-date and suggestions for better resources are welcome.